source/examples/vue/six-app/package.json

```diff
@@ -1,29 +1,36 @@
 {
-  "name": "six-app",
-  "version": "2.1.0",
+  "name": "@six-group/ui-library-vue-app",
+  "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "npm run serve",
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "clean": "rimraf dist"
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "format": "prettier --write src/"
   },
   "dependencies": {
-    "@six-group/ui-library": "2.0.0",
-    "@six-group/ui-library-vue": "2.0.0",
-    "core-js": "^3.6.5",
-    "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0"
+    "@six-group/ui-library": "file:../../../packages/ui-library",
+    "@six-group/ui-library-vue": "file:../../../packages/ui-library-vue",
+    "vue": "^3.2.47",
+    "vue-router": "^4.1.6"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-router": "~4.5.0",
-    "@vue/cli-plugin-typescript": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
-    "@vue/compiler-sfc": "^3.0.0",
-    "rimraf": "^3.0.2",
-    "sass": "^1.36.0",
-    "sass-loader": "^8.0.2",
-    "typescript": "^4.3.5"
+    "@rushstack/eslint-patch": "^1.2.0",
```

Comment:
This probably needs some explanation in a readme and according to the issue in eslint this has been introduced: https://github.com/eslint/eslint/issues/3458

Reply:
This is included in the default `package.json` created by the vue scaffolding with `npm init vue@latest` and I don't really know what the implications are.

Reply:
Hmm I usually create vue projects with the create-vite scaffolding https://vitejs.dev/guide/#scaffolding-your-first-vite-project and there I haven't come across this package. Since we are using vite, maybe we should switch it to that setup.

Here are the two stackblitz examples for vite projects with vue and vue-ts
https://stackblitz.com/edit/vitejs-vite-uxwvzm?file=package.json&terminal=dev
https://stackblitz.com/edit/vitejs-vite-bvz46r?file=package.json&terminal=dev

Reply:
I would suggest to use the setup that is officially recommended by vue, see https://vuejs.org/guide/quick-start.html.